modules/cloud-conf-shiftstack-reference.adoc

```diff
@@ -0,0 +1,245 @@
+//  Module included in the following assemblies:
```

Comment:
@MaysaMacedo @stephenfin I'm trying to figure out which cloud conf options are supported in 4.9 (hopefully extending into 4.11) here. Any immediate thoughts?

Reply:
The set of options that cloud provider accepts is the same in 4.8-4.11. The question might be what we actually support.